static_src/test/unit/components/service_plan.spec.jsx

```diff
@@ -55,3 +55,56 @@ describe('<ServicePlan />', () => {
     expect(props.onAddInstance.calledWith(props.plan.guid)).toBe(true);
   });
 });
+
+describe('<ServicePlan with multi param plan />', () => {
+  it('renders an action button for cdn-route', () => {
```

Comment:
I think all of these test cases could be collapsed into a single `it` block with a loop, almost all of the code is identical except for the names for the services.

`it('renders a link to documentation')` or something similar.